src/ugeneui/src/project_support/ProjectImpl.cpp

```diff
@@ -229,6 +230,21 @@ void ProjectImpl::sl_onObjectRelationChanged(const QList<GObjectRelation> &previ
     }
 }
 
+void ProjectImpl::sl_objectLoadedStateChanged() {
+    Document *doc = qobject_cast<Document *>(sender());
+    CHECK(doc != nullptr, )
```

Comment:
Not a blocker:
Can be done in  1 line instead of 4:
CHECK( doc != nullptr && !doc.isLoaded())